sdb/commands/help.py

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
When I pipe the output of `help` into `head`, it results in a crash:
```
> help | head
head: Return the first N objects passed in the pipeline
type: Return the type of the object passed
tail: Return the last N objects passed in the pipeline
echo: This is the superclass of all SDB command classes.
exit: Exit the application
stacks: This is the superclass of all SDB command classes.
filter: Returns a list of objects matching the given expression
pretty_print: This is the superclass of all SDB command classes.
member: This is the superclass of all SDB command classes.
cast: This is the superclass of all SDB command classes.
walk: This is the superclass of all SDB command classes.
pyfilter: This is the superclass of all SDB command classes.
help: Print command usage
count: Return the number of objects passed in the pipeline
address: This is the superclass of all SDB command classes.
slabs: A Locator is a command that locates objects of a given type.
dmesg: A Locator is a command that locates objects of a given type.
spl_kmem_caches: A Locator is a command that locates objects of a given type.
spl_list: A walker is a command that is designed to iterate over data
avl: walk avl tree
arc: A Locator is a command that locates objects of a given type.
metaslab: A Locator is a command that locates objects of a given type.
vdev: A Locator is a command that locates objects of a given type.
spa: A Locator is a command that locates objects of a given type.
zfs_dbgmsg: A Locator is a command that locates objects of a given type.
Traceback (most recent call last):
  File "/usr/local/bin/sdb", line 11, in <module>
    load_entry_point('sdb==0.1.0', 'console_scripts', 'sdb')()
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/internal/cli.py", line 216, in main
    repl.start_session()
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/internal/repl.py", line 114, in start_session
    _ = self.eval_cmd(line)
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/internal/repl.py", line 87, in eval_cmd
    for obj in objs:
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/__init__.py", line 180, in invoke
    yield from execute_pipeline(prog, first_input, pipeline)
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/__init__.py", line 77, in execute_pipeline
    yield from pipeline[-1].call(this_input)
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/commands/head.py", line 40, in call
    for obj in objs:
  File "/usr/local/lib/python3.6/dist-packages/sdb-0.1.0-py3.6.egg/sdb/__init__.py", line 77, in execute_pipeline
    yield from pipeline[-1].call(this_input)
TypeError: 'NoneType' object is not iterable
```
I'm not sure if this is a bug in this new command, or in the "infrastructure" of SDB.